src/components/CodeArea/CodeArea.tsx

```diff
@@ -59,7 +64,11 @@ const CodeArea: FC<ICodeAreaProps> = ({
                         spellCheck={false}
                         value={currentValue}
                         disabled={disabled}
-                    />
+                    >
+                    </TextAreaStyled>
+                    <MaximizeButtonStyled>
```

Comment:
i think maybe the PR is a work in progress as i now see you are missing the `hasMaxHeight` boolean we talked about earlier. Ping me on Slack when you update. TY